src/packages/database/query/index.js

```diff
@@ -164,10 +165,11 @@ class Query<+T: any> extends Promise {
         this.snapshots = this.snapshots
           .filter(([method]) => method !== 'orderBy')
```

Comment:
Just noticed this line has to change to `.filter(([method]) => method !== 'orderByRaw')`.